### PR TITLE
Create arrays_ali_geris.py

### PR DESCRIPTION
**Pull request by gerisali**
## Checklist
- [ ] I have read the [CONTRIBUTING]
- [ ] I have performed a self-review of my own code
- [ ] I have run the code locally and it works as expected
- [ ] I have commented my code, particularly in hard-to-understand areas

### CODE REVIEW

File: Week04/arrays_ali_geris.py [hunk 1]
```diff
@@ -0,0 +1,39 @@
+import numpy as np
+
+
+def replace_center_with_minus_one(d, n, m):
+    if d <= 0 or n <= 0 or m <= 0:
+        raise ValueError("Invalid input parameters.")
+    if m > n:
+        raise ValueError("m cannot be greater than n")
+
+    arr = np.random.randint(0, 10 ** d, size=(n, n))
```
Comment (canbula) [inline Week04/arrays_ali_geris.py:10]: Be careful about the range of the random integers. It does not start from `0`.

**Review by canbula — CHANGES_REQUESTED** [review]
See my comment below.

**Review by canbula — CHANGES_REQUESTED** [review]
It must depend on `n`.

File: Week04/arrays_ali_geris.py [hunk 2]
```diff
@@ -0,0 +1,43 @@
+import numpy as np
+
+
+def replace_center_with_minus_one(d, n, m):
+    if d <= 0 or n <= 0 or m <= 0:
+        raise ValueError("Invalid input parameters.")
+    if m > n:
+        raise ValueError("m cannot be greater than n")
+
+    low = 100
```
Comment (canbula) [inline Week04/arrays_ali_geris.py:10]: `low` must be dependent  on `d`.

**Review by canbula — CHANGES_REQUESTED** [review]
See my comment below.

**Comment by gerisali** [comment]
Code lines updated.

**Review by canbula — APPROVED** [review]
👍

**Pull request merged**